.travis.yml

```diff
@@ -15,6 +15,8 @@ android:
     - tools
     - build-tools-26.0.1
```

Comment:
Can it probably be removed?

Reply:
Unfortunately it needs both, or it starts complaining about missing licenses...